src/legacy/core_plugins/vis_type_table/public/paginated_table/rows.js

```diff
@@ -70,9 +71,17 @@ export function KbnRows($compile) {
           $cell = $cellContent = createCell();
         }
 
+        const options = {
```

Comment:
Conceptual question, instead of increase number of parsedUrl object  copies we should think how to remove duplicates of it.